### ISSUE TITLE
Support PHP7 or changeable PHP Version

### ISSUE THREAD

**Issue by timohund** [Tuurlijk/TYPO3.Packer#7]
Thx for your awesome work on this box. I think it would be nice to have php7 or even better to be able to use different PHP Versions in parallel.

**Comment by Tuurlijk** [comment]
It is currently implemented using phpbrew. https://github.com/phpbrew/phpbrew

Currently available versions: https://github.com/Tuurlijk/TYPO3.Packer/blob/master/ansible/configuration/Development/php.yml#L56

But only on the dev and review boxen.

I'll look into getting 7 up and running

**Comment by Tuurlijk** [comment]
Fixed in https://github.com/Tuurlijk/TYPO3.Packer/commit/d845193e9c9e7c208062b15103d98a43ad8f2791
- http://php55.local.typo3.org/
- http://php56.local.typo3.org/
- http://php70.local.typo3.org/
- http://hhvm.local.typo3.org/

Versions used: https://github.com/Tuurlijk/TYPO3.Packer/blob/master/ansible/configuration/Development/php.yml#L56

**Comment by timohund** [comment]
@Tuurlijk Nice, thx!
